browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchRequest.java

```diff
@@ -1,39 +1,58 @@
 package com.braintreepayments.api;
 
 import android.net.Uri;
+import android.util.Base64;
 
 import androidx.annotation.NonNull;
 import androidx.annotation.VisibleForTesting;
 
 import org.json.JSONException;
 import org.json.JSONObject;
 
+import java.nio.charset.StandardCharsets;
+
+// Links
+// Base64 Encode a String in Android: https://stackoverflow.com/a/7360440
+
+// TODO: consider encryption
+// Ref: https://medium.com/fw-engineering/sharedpreferences-and-android-keystore-c4eac3373ac7
+
+// TODO: Rename to `BrowserSwitchStartRequest` and remove `BrowserSwitchOptions` in favor this class
```

Comment:
Should we remove these TODOs before merging the PR in?

Reply:
I agree, we may be able to convert these to JIRA tickets.